.github/workflows/bot.yml

```diff
@@ -125,4 +125,8 @@ jobs:
         run: |
           HUDI_VERSION=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
           ./packaging/bundle-validation/ci_run.sh $HUDI_VERSION
-
+      - name: Platform Service Build and Test
```

Comment:
this can be moved to a separate CI job in parallel to bundle validation. this does not require matrix of different spark/flink versions so it should just run once

Reply:
fix